quixstreams/state/rocksdb/windowed/partition.py

```diff
@@ -81,19 +72,9 @@ def write(
     ):
         batch = WriteBatch(raw_mode=True)
 
-        if latest_timestamp_ms is not None:
```

Comment:
`latest_timestamp_ms` is not used anymore

Reply:
The whole `write()` method looks like a leftover of the previous implementation.
We don't need to override it anymore